Application/src/model/DiagramCheck.java

```diff
@@ -1,57 +1,54 @@
 package model;
 
-import javafx.scene.control.Alert;
+import controller.Import;
+
 import java.util.Collection;
 
 /**
- * @version 0.1
+ * @version 1.0
  * @author Tim Jonasson & Kosara Golemshinska
+ * collaborator: Pontus Laestadius
  * @since 2017-10-10
  */
 
-public class DiagramCheck {
-	// Supported diagram types.
+class DiagramCheck {
+
+    // Supported diagram types.
+    private final static String[] allowedDiagrams = {"sequence_diagram", "class_diagram", "deployment_diagram"};
```

Comment:
you don't have to change this, but maybe add deployment_diagram later when we work on it? :)

Reply:
no, I think that's fine, we don't know what else we will have time for this sprint